src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Slider/SliderHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Xamarin.Platform.Handlers
+{
+	public partial class SliderHandler
+	{
+		public static PropertyMapper<ISlider, SliderHandler> SliderMapper = new PropertyMapper<ISlider, SliderHandler>(ViewHandler.ViewMapper)
+		{
+			[nameof(ISlider.Minimum)] = MapMinimum,
+			[nameof(ISlider.Maximum)] = MapMaximum,
+			[nameof(ISlider.Value)] = MapValue,
+			[nameof(ISlider.MinimumTrackColor)] = MapMinimumTrackColor,
+			[nameof(ISlider.MaximumTrackColor)] = MapMaximumTrackColor,
+			[nameof(ISlider.ThumbColor)] = MapThumbColor
+		};
+
+		public static void MapMinimum(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+			handler.TypedNativeView?.UpdateMinimum(slider);
+		}
+
+		public static void MapMaximum(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+			handler.TypedNativeView?.UpdateMaximum(slider);
+		}
+
+		public static void MapValue(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+			handler.TypedNativeView?.UpdateValue(slider);
+		}
+
+		public static void MapMinimumTrackColor(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+			handler.TypedNativeView?.UpdateMinimumTrackColor(handler, slider);
```

Comment:
At this point we also pass the handler as a parameter. But... why?. Well, we need to use default colors in the extension methods. The colors have been defined in the handler and set when initializing the native view. There are more ways to achieve the same result, feedback?